src/compiler.ts

```diff
@@ -7220,18 +7220,21 @@ export class Compiler extends DiagnosticEmitter {
     var stub = original.varargsStub;
     if (stub) return stub;
 
+    // this stub relies on the existence of the argumentsLength global
+    this.ensureArgumentsLength();
```

Comment:
It seems that we have introduced a way to create this varargs stub method even when it isn't called- which means that the `argumentsLength` global won't be set.

Certainly if we get here this variable must be defined, so an ensure here makes sense anyway.